R/build_lookup.R

```diff
@@ -62,6 +62,39 @@ parse_imgt_fasta <- function(infile) {
   imgt_list
 }
 
+#' Extract gene names from a reference CSV for a given species
+#'
+#' `parse_mixcr_csv()` generates mixcr gene name data for a given species.
+#' @param species A string specifying the species, one of "human", "mouse",
+#' or "rhesus".
+#'
+#' @return A character vector of gene names.
+#' @export
+#' @keywords internal
+#' @examples
+#' parse_mixcr_csv("human")
+parse_mixcr_csv <- function(species) {
```

Comment:
```suggestion
parse_mixcr_csv <- function(data_dir) {
```
Instead of a species subfolder name, the input should just be a path to where a pile of mixcr CSVs are (similar to `extract_imgt_genes()`)

Reply:
https://github.com/seshadrilab/tcrconvertr/blob/7d3cb85934f6e058f528fb1d9f2036fa4c62a054/R/build_lookup.R#L76-L80

Changed to accept a path to a directory with mixcr CSVs